releases.yaml

```diff
@@ -1,5 +1,37 @@
 ---
 releases:
+  22.09-alpha1:
+    releaseDate: 2022-09-07
+    description: Second release focusing on security and OpenShift support. Will be replaced by release 22.09
```

Comment:
Maybe something like "Alpha 1 of release 22.09 - release focus is OpenShift support and security."

Reply:
Updated, please check